### ISSUE TITLE
jsonGraphqlExpress is undefined

### ISSUE THREAD

**Issue by terion-name** [marmelab/json-graphql-server#44]
jsonGraphqlExpress is not present in node bundle in npm

```
import { jsonGraphqlExpress } from 'json-graphql-server';
console.log(jsonGraphqlExpress); // undefined
```

**Comment by sehmaschine** [comment]
had the same issue and solved it like this:

```
import jsonGraphqlExpress from 'json-graphql-server';
```

**Comment by djhi** [comment]
Thanks @sehmaschine! Can you make a PR for the README ? :bowing_man:

**Comment by fzaninotto** [comment]
Fixed by #50
